### DISCUSSION TITLE
Vratakathā - Discipline or Genre

### DISCUSSION THREAD

**Discussion by chakrabortydeepro** [PanditProject/Website#47] [Help]
@ybronner Hello,
I wanted to include certain vratakathā texts. They are usually attributed to certain purāṇas. They are also included in the Paddhati texts. For example, Anantavrata is a paddhati text which includes Anantavratakathā which in turn is attributed to a the Bhaviṣyapurāṇa. 

I was wondering if _Purāṇa_ is the discipline and _vratakathā_ is a genre or _vratakathā_ itself is a discipline and _kathā_ is the genre.

**Comment by ybronner** [comment]
These things are tricky, so it is hard to know, but the important thing is to get all the labels for future search purposes. My hunch is that Purāṇa is the discipline and vratakathā is a genre, and that so is Paddhati. I doubt that kathā should be used as a separate genre in this sense. If we give Vratakathā and Paddhati as genres we will have covered our bases. More problematic in my mind is how to encode the relationship between the specific purāṇa (in this case Bhaviṣyapurāṇa) and the Anantavrata. Any thought on what the relationship is?

**Comment by wujastyk** [comment]
In case it helps, the purāṇa/vrata relationship is treated by traditional librarianship as "analytical cataloguing".  This is the same technique as is used for cataloguing the individual chapters in a conference proceeding, for example.  One item is a member of another item.  There's a lot of writing about analytical records and analytical cataloguing.  I don't know if these approaches can be used or would be appropriate in PP.

**Comment by chakrabortydeepro** [comment]
Thanks for your answers, Prof. Bronner and Prof. Wujastyk. @ybronner It would be useful then if you create a new genre for _vratakathā_. 
I am not sure if every _vratakathā_ texts really appear in the main Purāṇas as per the attributions. I need to ask a Purāṇa-expert.

**Reply by ybronner** [reply]
Vratakathā is added. I'll look into analytical cataloging. But I should say that we currently have a "Part of (work)" field, if work a is part of a larger work b (I don't know if this applies here). I should also mention that we are about to overhaul our work-to-work relationship scheme in such a way that would make it much easier to add a relationship type. More on this later, as part of our move to Drupal 9.

**Comment by wujastyk** [comment]
"part of work" should meet this need very well, I think.  
About vratas, kathās, mālās, and so on, my impression when cataloguing hundreds of these during Wellcome days is that it's a pretty fluid series of works.  Some really do belong in a Purāṇa, others are independent but *say* that they're part of a Purāṇa.  Scores of works attribute themselves to a fantasy Rudrayāmala.   I don't quite know what this implies for PP, except that we shouldn't expect to fit all such devotional works into a clear framework.  
Prof. Parameswar Aithal dedicated much work to cataloguing stotras, if I recall.  Study of his catalogues might throw some light on all this.
Best
Dominik

**Comment by chakrabortydeepro** [comment]
Thanks, Prof. Wujastyk for shedding some light on this.
